lib/computeStatistics.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+var Cesium = require('cesium');
+var isDataUri = require('./isDataUri');
+var PrimitiveHelpers = require('./PrimitiveHelpers');
+
+var defined = Cesium.defined;
+
+module.exports = {
+    getStatistics : getStatistics,
+    printStatistics : printStatistics
+};
+
+/**
+ * Creates an object type for glTF Statistics.
+ *
+ * @property {Number} buffersSizeInBytes The total size of all the buffers.
+ * @property {Number} numberOfImages The number of images part of the assest.
```

Comment:
`The number of images in the asset` is better. Also fix spellings - `assest` appears a few times.